index.html

```diff
@@ -126,29 +133,32 @@ <h1 class="project-title">Multi-Post<br>Stories</h1>
          </section> 
 
          <section class="about">
-            <div class="intro">
-                <h1>About<br>Myself</h1>
-                <p>I can help you build a product , feature or website Look through some of my work and experience! If
-                    you like what you see and have a project you need coded,
-                    don’t hestiate to contact me.
-                </p>
-            </div>
-                <!-- social medials lins -->
-            <div class="social-medials">
-                <p>LET’S CONNECT</p>
-                <ul class="social-icons">
-                    <li><a href="https://github.com/Ridwanullahi-code"><img src="images/github.png" alt=""></a></li>
-                    <li><a href="https://www.linkedin.com/in/ajayi-ridwan-2a1515199/"><img src="images/Vector.png" alt=""></a></li>
-                    <li><a href="https://www.linkedin.com/in/ajayi-ridwan-2a1515199/"><img src="images/Linkedin.png" alt=""></a></li>
-                    <li><a href="https://twitter.com/Ridwanullahi22"><img src="images/twitter.png" alt=""></a></li>
-                    <li><a href="https://github.com/Ridwanullahi-code"><img src="images/medium.png" alt=""></a></li>
-                    
-                </ul>      
-            </div>
-            <div>
-                <!-- button to download resume -->
-                <a class="button btn" href="">Get my resume</a>
+            <div class="about-summary">
+                <div class="intro">
+                    <h1>About<br>Myself</h1>
+                    <p>I can help you build a product , feature or website Look through some of my work and experience! If
+                        you like what you see and have a project you need coded,
+                        don’t hestiate to contact me.
+                    </p>
+                </div>
+                    <!-- social medials lins -->
+                <div class="social-medials">
+                    <p>LET’S CONNECT</p>
+                    <ul class="social-icons">
+                        <li><a href="https://github.com/Ridwanullahi-code"><img src="images/github.png" alt=""></a></li>
+                        <li><a href="https://www.linkedin.com/in/ajayi-ridwan-2a1515199/"><img src="images/Vector.png" alt=""></a></li>
+                        <li><a href="https://www.linkedin.com/in/ajayi-ridwan-2a1515199/"><img src="images/Linkedin.png" alt=""></a></li>
+                        <li><a href="https://twitter.com/Ridwanullahi22"><img src="images/twitter.png" alt=""></a></li>
+                        <li><a href="https://github.com/Ridwanullahi-code"><img src="images/medium.png" alt=""></a></li>
+                        
+                    </ul>      
+                </div>
+                <div>
+                    <!-- button to download resume -->
+                    <a class="button btn" href="">Get my resume</a>
+                </div>
             </div>
```

Comment:
- [ ] This section should be a static HTML with css styling and not an image kindly remove the image and add a list using `<ul>`

![image](https://user-images.githubusercontent.com/89282221/187988099-99661349-0962-4f30-ba7c-ac4c65fa9816.png)